css/src/editor.scss

```diff
@@ -167,15 +168,19 @@ div[class^="wp-block-block-lab-"] {
       font-size: 1em;
       color: rgba(0, 0, 0, 0.8);
     }
-  }
 
-  /* FetchInput component */
-  .bl-fetch-input {
-    max-width: $input-width;
+    /* Override a max-width: 25rem style rule in Core that can prevent displaying at the selected with, like 75% */
+    .components-select-control__input {
+      max-width: unset;
+    }
   }
 
   /* Rich Text Control Component */
   .block-lab-rich-text-control {
+    .components-base-control__field {
+      margin-top: 20px;
```

Comment:
This addresses https://github.com/getblocklab/block-lab/issues/460.

# Before 

<img width="786" alt="before-little-or-no-space" src="https://user-images.githubusercontent.com/4063887/68077873-5a627d00-fd91-11e9-80a3-470eae7dde38.png">

# After

<img width="684" alt="after-more-space" src="https://user-images.githubusercontent.com/4063887/68077869-4fa7e800-fd91-11e9-937b-6acf37e08f0b.png">